backend/migrations/versions/799a2343955e_fix_moduletype_enum.py

```
@@ -0,0 +1,118 @@
+    "PYTHON_GUIDE",
+    "RECOGNITION_GUIDE",
+    "QUIZ",
+    "CHALLENGE",
+    "CHALLENGE_SOLUTION",
+    "BONUS_CHALLENGE",
+    "BONUS_SOLUTION",
+    name="moduletype",
+)
+
+
+def upgrade():
```

Ruff (0.8.2)

Missing return type annotation for public function `upgrade`

Add return type annotation: `None`

(ANN201)